Makefile

```diff
@@ -51,9 +51,13 @@ validate-docs:
 	towncrier --draft --version preview
```

Comment:
At least one more `towncrier` invocation that requires a `build` keyword. Maybe a `git grep` to double-check that there are no more?